src/styles/_onboarding.scss

```diff
@@ -7,70 +7,70 @@
   background-color: $onboarding-bg-blue;
   z-index: 90;
   text-align: center;
-  padding-top: 10vh;
+  padding: 0;
   color: white;
 
-  @media ($aboveMin) {
-    .ob-main {
-      height: 50vh;
+  .ob-main, .ob {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+  }
 
-      .login-btn {
-        margin-top: 15px;
-      }
-    }
+  .ob-main {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+  }
 
-    .ob-nav {
-      height: 15vh;
+  .ob {
+    width: 100%;
+    height: 100%;
+    max-height: 500px;
+    max-width: 500px;
+    padding: 20px;
+    justify-content: space-around;
+    overflow: auto;
+
+    @media (max-height: 500px) { 
```

Comment:
This might be worth another variable, like the other resizing @media queries I made earlier.